src/docs/binding-structs.md

```diff
@@ -0,0 +1,100 @@
+# Binding Structs
+
+Structs in Swift are value types that can have a number of forms each of which requires a certain amount of consideration: frozen/non-frozen and blitable and non-blitable.
+
+Structs that are frozen are guaranteed not to change. They have a fixed layout and are passed by value lowered up until the point when the size of the struct exceeds the number of registers dedicated for value types in Swift (currently 4 and unlikely to change).
```

Comment:
Frozen structs with `weak` fields are passed by value. I don't believe that non-bitwise-copyable structs are passed by value, but I may need to check.

```suggestion
Structs that are frozen are guaranteed not to change. They have a fixed layout and can be passed by value lowered up until the point when the size of the struct exceeds the number of registers dedicated for value types in Swift (currently 4 and unlikely to change).
```